templates/test/ci/prow-clusterclass-ci-default/patches.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
   name: ${CLUSTER_CLASS_NAME}
+  labels:
+    # TODO: do labels work with clusterclass?
```

Comment:
is this still TODO?

Reply:
I'm inclined to say yes since the tests are passing but I'll double check.

Reply:
Seems like it doesn't work with clusterclass. I think we can just add the labels directly to the prow-topology template.